templates/flamegraph.mustache

```diff
@@ -65,6 +66,10 @@
             <th>{{#str}} field_request, tool_excimer {{/str}}</th>
             <td>{{request}}</td>
         </tr>
+        <tr>
+            <th>{{#str}} field_pathinfo, tool_excimer {{/str}}</th>
```

Comment:
same as the table, lets just have a single 'request url' which all the same reused logic as in the first table